tags

```diff
@@ -0,0 +1,37 @@
+Post243ToMaster
+Post250ToMaster
+Post251ToMaster
+Pre243ToMaster
+Pre244ToMaster
+Pre250ToMaster
+Pre251ToMaster
+last-online-docs
+s1.8.4
+s1.8.5
+s1.9.0
+s1.9.1
+s1.9.2
+s1.9.3
+s1.9.4
+s2.0.0
+s2.0.1
+s2.1.1
+s2.1.2
+s2.2.0
+s2.3.0
+s2.4.1
+s2.4.2
+s2.4.3
+s2.4.4
+s2.4.5
+s2.5.0
+s2.5.1
+s2.6.0
+sRecurrence
+stable
+v2.5.0.beta1
+v2.5.0.beta2
+v2.5.0.beta3
+v2.5.1.beta1
+
```

Comment:
Unclear what this file is for, but I suppose we should trash it?

Reply:
Oops, yes, that was a `git add .` mistake!